src/config/analyticsRoutes.ts

```diff
@@ -0,0 +1,271 @@
+import type { AnalyticsEvent } from "config/analyticsEvents";
+import { logger } from "config/logger";
+import {
+  AUTH_STACK_ROUTES,
+  BUY_XLM_ROUTES,
+  MAIN_TAB_ROUTES,
+  MANAGE_ASSETS_ROUTES,
+  MANAGE_WALLETS_ROUTES,
+  ROOT_NAVIGATOR_ROUTES,
+  SEND_PAYMENT_ROUTES,
+  SETTINGS_ROUTES,
+  SWAP_ROUTES,
+} from "config/routes";
+
+/**
+ * Analytics Route Mapping
+ *
+ * Maps React Navigation routes to analytics events using automatic transformation.
+ *
+ * ## How it works:
+ * 1. **Automatic**: "WelcomeScreen" → "loaded screen: welcome"
+ * 2. **Automatic**: "SettingsScreen" → "loaded screen: settings"
+ * 3. **Override only**: Special cases that don't follow the pattern
+ *
+ * ## Adding new screens:
+ * 1. Add route to routes.ts (normal process)
+ * 2. Analytics tracking works automatically ✨
+ * 3. Only add override if the auto-generated name is wrong
+ */
```

Comment:
I like that we are automating the route events based on the route name, but this also means we should be more mindful from now on when editing route names. Wdyt?
Would you mind adding comments on the `routes.ts` file to make it very explicit that the route name is also being used for analytics? So we prevent analytics disruption